GoogleForCreators/web-stories-wp
# Template Updates: Fitness

## Context

Part of 1.7 release is updating existing templates to avoid custom animations and unnecessary prepublish checklist suggestions.

## Summary

Fitness Template revisions 
- Remove custom animations and replace with effects available in editor 
- Add CTA as sticker on page 7

## Relevant Technical Choices

Used my own judgement for which direction animations should come from. 

## To-do


## User-facing changes

See figma for revised template: https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories-Post-Stable?node-id=7357%3A546501
List of changes can be found here: #6974 

**Before**
https://user-images.githubusercontent.com/10720454/114282115-d46f4a80-99f6-11eb-8952-91e41bac6356.mp4


**After**
https://user-images.githubusercontent.com/10720454/114281378-e8b14880-99f2-11eb-943d-a0dec4a90162.mp4


## Testing Instructions

- use template from dashboard / explore templates (fitness template) and see that there are no more custom animations on any elements and that the little circle on page 7 is a sticker. 

### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?

No

### Does this PR change what data or activity we track or use?

No

### Does this PR have a legal-related impact?

No

## Checklist




---



Fixes #6974
